pages/common/touch.md

```diff
@@ -1,24 +1,20 @@
 # touch
 
-> Change a file access and modification times (atime, mtime).
+> Create files and set access/modification times.
 > More information: <https://www.gnu.org/software/coreutils/touch>.
 
-- Create a new empty file(s) or change the times for existing file(s) to current time:
+- Create specific files:
 
-`touch {{path/to/file}}`
+`touch {{path/to/file1 path/to/file2 ...}}`
 
-- Set the times on a file to a specific date and time:
+- Set the file access/modification times to the current one:
 
-`touch -t {{YYYYMMDDHHMM.SS}} {{path/to/file}}`
+`touch --no-create --time={{access|modify}} {{path/to/file1 path/to/file2 ...}}`
 
-- Set the time on a file to one hour in the past:
+- Set the file [t]ime to a specific value:
 
-`touch -d "{{-1 hour}}" {{path/to/file}}`
+`touch --no-create --time={{access|modify}} -t {{YYYYMMDDHHMM.SS}} {{path/to/file1 path/to/file2 ...}}`
 
-- Use the times from a file to set the times on a second file:
+- Set file time of a specific file to the time of another file:
 
-`touch -r {{path/to/file1}} {{path/to/file2}}`
-
-- Create multiple files:
-
-`touch {{path/to/file{1,2,3}.txt}}`
+`touch --no-create --time={{mode}} --reference={{path/to/file}} {{path/to/file1 path/to/file2 ...}}`
```

Comment:
This way is more clear
```suggestion
`touch --no-create --time={{access|modify}} --reference={{path/to/file}} {{path/to/file1 path/to/file2 ...}}`
```

Reply:
@navarroaxel, should I always inline all values for options? What about to do it only when value count is small such as here (just two choices)?

Reply:
This really depends. There are mostly three factors here: The number of options, how intuitive the options are, and how quickly the options can be found in e.g. the man page.

For few options, we can almost always inline them.
If you have many, but intuitive options (like `read`, `write`, `execute`) which everyone knows, we can be generic.
And if the options, can't be easily found somewhere we should also inline them.

But this is highly dependent on the situation. Just think about how it would be the easiest for the user.